python_ta/debug/accumulation_table.py

```diff
@@ -64,50 +76,73 @@ class AccumulationTable:
     accumulation variables during each iteration in a for or while loop
 
     Instance attributes:
-        loop_accumulators: a mapping between the accumulation variables
```

Comment:
Keep the documentation for this attribute and `loop_variables`, but add a phrase to indicate that they are read-only and only available when the context manager is wrapping a single loop